requirements.txt

```
@@ -0,0 +1,4 @@
+uvicorn == 0.13.2
+fastapi == 0.63.0
+requests == 2.25.1
+cryptography == 3.3.1
```

Use `==` without spaces. This is the convention with `requirements.txt` files.

```suggestion
uvicorn==0.13.2
fastapi==0.63.0
requests==2.25.1
cryptography==3.3.1
```